lib/lita/adapters/slack/message_handler.rb

```diff
@@ -160,6 +162,26 @@ def handle_message
           dispatch_message(user)
         end
 
+        def handle_reaction
+          log.debug "#{type} event received from Slack"
+
+          # find or create user
+          user = User.find_by_id(data["user"]) || User.create(data["user"])
+
+          # avoid processing reactions added/removed by self
+          return if from_self?(user)
+
+          # build a payload following slack convention for reactions
+          payload = { user: user, name: data["reaction"], item: data["item"], event_ts: data["event_ts"] }
+
+          # trigger the appropriate slack reaction event
+          if type == "reaction_added"
```

Comment:
In case you are absolutely sure about `type` being either `reaction_added` or `reaction_removed`, you can do this instead:

```ruby
robot.trigger("slack_#{type}".to_sym, payload)
```